automated/linux/mmtests/json-to-lava.py

```diff
@@ -23,7 +24,13 @@ def main(args):
                 values = op_data["Values"]
                 samples = op_data["SampleNrs"]
                 for val, sample in zip(values, samples):
-                    print(f"{module_name}-{test_name}-{op}-{i}-{sample} pass {val}")
+                    print(f"{module_name};{op};{i};{sample} pass {val}")
+        if "_Cmd" in module:
+            string = ""
+            for i in range(len(module['_Cmd'])):
+                string = string + module['_Cmd'][i]
+            b64string = b64encode(bytes(string,'utf-8')).decode('utf-8')
+            print(f"{module_name};_Cmd;{0};{b64string} pass {0}")
```

Comment:
Is there any better way to read "b64string" in human readable format ?

  dbench4tput;_Cmd;0;b'ZGJlbmNoIC1EIC9ob2 ... lLXRpbWUgMQo=' pass 0

Reply:
That's the best way I thought of.
I know it is not intelligible by humans, but the spaces in the string will break the conversion, and b64 is a great way to not have any weird characters or whitespaces breaking the syntax